host/xtest/Makefile

```diff
@@ -33,8 +33,12 @@ define cc-bits
 $(if $(filter arm, $(1)),32,$(if $(filter aarch64, $(1)),64,unknown-arch))
 endef
 
-# OpenSSL is used by GP tests series 8500 and Mbed TLS test 8103
-ifneq ($(CFG_GP_PACKAGE_PATH)$(filter y,$(CFG_TA_MBEDTLS)),)
+# OpenSSL is used by:
+# - GP tests series 8500
+# - Mbed TLS test 8103
+# - UUID tests 1027 and 1028
```

Comment:
Minor: "User/group login tests 1027 and 1028"

Reply:
Done.